src/lib.rs

```diff
@@ -33,28 +42,21 @@ pub struct Params {
     #[structopt(short = "v", long, default_value = "5")]
     num_vertices_arc: usize,
 
-    /// Number of decimal places in the resulting output (GeoJSON) files.
+    /// Number of decimal places in the resulting output (GeoJSON).
     /// Set to 6 by default. Larger numbers mean more precision but
     /// larger file sizes.
     #[structopt(short, long, default_value = "6")]
     precision: usize,
+
+    /// Is the data projected?
+    /// False by default.
+    #[structopt(long, parse(try_from_str), default_value = "false")]
```

Comment:
As long as you're OK with `--projected` to enable this, I think you can just do `#[structopt(long)] projected: bool`. You don't need your own string->bool parser. If not specified, structopt will use the `std::default::Default` of the type. And for bools, that's `false`

Reply:
I think I tried something similar but after looking at `structopt` docs and trial and error this seems to be what worked. I would love to get it more concise though.

Reply:
OK I stand corrected, you're right on this Dustin. Happy to say, this more concise version seems to work:

```
    #[structopt(long)]

```